src/main/java/io/vertx/core/impl/resolver/DnsResolverProvider.java

```diff
@@ -85,7 +85,7 @@ public DnsResolverProvider(VertxImpl vertx, AddressResolverOptions options) {
         }
       }
     } else {
-      DnsServerAddressStream stream = DnsServerAddresses.defaultAddresses().stream();
+      DnsServerAddressStream stream = DefaultDnsServerAddressStreamProvider.defaultAddresses().stream();
```

Comment:
why this change ?

Reply:
since `DnsServerAddresses#defaultAddresses()` is deprecated

Reply:
it is recommended to use  `DefaultDnsServerAddressStreamProvider#defaultAddresses()`

Reply:
ok